lib/api/src/grpc/proto/collections.proto

```diff
@@ -50,6 +50,14 @@ message GetCollectionInfoRequest {
   string collection_name = 1; // Name of the collection
 }
 
+message CollectionExistsRequest {
+  string collection_name = 1;
+}
+
+message CollectionExistsResponse {
+  bool exists = 1;
+}
```

Comment:
Just for consistency, we should also add timing in grpc, same as we do in http

Reply:
thank you, just did this